File: webHomeWork/projectX/src/main/java/com/homework/models/JsonPackage.java
package com.homework.models;

/**
 * Created by mikhail on 13.09.16.
 */
public class JsonPackage {
    private String action;
    private String[] docsName;
    private String[] textInDocs;

    public JsonPackage(String action, String[] docsName, String[] textInDocs) {
        this.action = action;
        this.docsName = docsName;
        this.textInDocs = textInDocs;
    }

    public JsonPackage(String action) {
        this.action = action;
    }
}

File: webHomeWork/kotlinProjectX/src/main/kotlin/homework/com/models/JsonPackage.kt
package com.homework.models

/**
 * Created by mikhail on 13.09.16.
 */
class JsonPackage  {
    private val action:String
    private val docsName:Array<String>
    private val textInDocs:Array<String>
    constructor(action:String, docsName:Array<String>, textInDocs:Array<String>) {
        this.action = action
        this.docsName = docsName
        this.textInDocs = textInDocs
    }
    constructor(action:String) {
        this.action = action
    }
}
}
